config/openshift/base/operator.yaml

```diff
@@ -48,6 +48,8 @@ spec:
             drop:
               - ALL
         env:
+        - name: KUBERNETES_MIN_VERSION
+          value: "v1.0"
```

Comment:
Should we also for k8s specific platform https://github.com/tektoncd/operator/blob/main/config/kubernetes/base/operator.yaml

Reply:
I think there is a bit a consensus to not do that upstream for now,  I would say